tc/core/polyhedral/cuda/mapped_scop.h

```diff
@@ -186,6 +186,14 @@ class MappedScop {
       size_t nChildren,
       bool hasOuterSequentialMember);
 
+  // Extract a mapping from the domain elements active at "tree"
+  // to the thread identifiers, where all branches in "tree"
+  // are assumed to have been mapped to thread identifiers.
+  // "nThread" is the number of thread identifiers.
+  // The result lives in a space of the form block[x, ...].
+  isl::multi_union_pw_aff threadMappingSchedule(
```

Comment:
Same comment as in the previous commit, there is no schedule involved here so I would call this `threadMappingExpressions` or something along that vein.

Reply:
> Same comment as in the previous commit, there is no schedule involved here so I would ca
ll this `threadMappingExpressions` or something along that vein.

Do you mean that this is more like a "placement" than a "schedule"?
Usually, we don't make this distinction at this level.
I don't think "Expressions" adds any value.
The main point it to get the mapping to threads, so if you don't
like the "Schedule" bit, I'd just call it "threadMapping".

Reply:
Ah I see the terminology impedance mismatch. What ticked me off here is that I have built this mental model that functions involving the word schedule usually collect the mupa upt to the root. Similarly for "active elements", "active domain" and sometimes "domain" which I grew accustomed to seeing call `activeDomainPoints`. Should I revisit these expectations? threadMapping seems fine to me too.

Reply:
We represent the mapping as a `union_set` filter _and_ as a `multi_union_pw_aff`.  I actually use both in a function in the further commits blocked by this PR.  I wanted to make it clear which type is used for the mapping in a particular location, so I went for the explicit `Schedule` suffix.

>  that functions involving the word schedule usually collect the mupa upt to the root

Not necessarily, for example `infixScheduleMupa`.